app/models/room.rb

```diff
@@ -1,4 +1,9 @@
 class Room < ApplicationRecord
+
+  validates :capacity, presence: true
+  validates :floor, presence: true
+  validates :number, presence: true
```

Comment:
`floor` and `number` should validate non-negative. Can do it this way: `validates :floor, :numericality => { :greater_than_or_equal_to => 0 }`